src/harness/reference_models/antenna/antenna.py

```diff
@@ -99,7 +99,8 @@ def GetStandardAntennaGains(hor_dirs, ant_azimuth=None, ant_beamwidth=None, ant_
   is_scalar = np.isscalar(hor_dirs)
   hor_dirs = np.atleast_1d(hor_dirs)
 
-  if ant_beamwidth is None or ant_azimuth is None:
+  if (ant_beamwidth is None or ant_azimuth is None or
+      ant_beamwidth == 0 or ant_beamwidth == 360):
```

Comment:
Looks good to me! Just a small typo in docstring! **ant_beamwidth:  Antenna 3dB `curoff->cutoff` beamwidth (degrees)**

Reply:
Thanks I fixed the typo